background.js

```diff
@@ -1,29 +1,31 @@
 const urlRegex = new RegExp(/^https?:\/\/(?:[^./?#]+\.)?facebook\.com/);
-var pathVariable = '';
+var groupId = '';
 
 chrome.tabs.onUpdated.addListener(function (tabId , info) {
-    chrome.tabs.query({
-        active: true,
-        currentWindow: true
-    }, function (tabs) {
-        let url = tabs[0].url.substring(7);
-        let urlParts = url.split('/');
-        pathVariable = urlParts[3];
-    });
+    if (info.status === 'loading') {
+        chrome.tabs.query({
+            active: true,
+            currentWindow: true
+        }, function (tabs) {
+            let url = tabs[0].url.substring(7);
+            let urlParts = url.split('/');
+            groupId = urlParts[3];
+        });
+    }
 });
 chrome.browserAction.onClicked.addListener(function (tab) {
 
     function sendPosts(list) {
         for (let i = 0; i < list.length; i++) {
             var xhr = new XMLHttpRequest();
-            xhr.open("POST", "http://192.168.0.124:3000/savePosts/"+pathVariable, true);
+            xhr.open("POST", "http://192.168.0.124:3000/savePosts/"+groupId, true);
```

Comment:
String template will be good think in places like this, e.g.:
`xhr.open("POST", `http://(...)/${groupId}`, true);`

And should be used one type of quotemark (') or (") in project